packages/material-ui-lab/src/DatePicker/index.ts

```diff
@@ -1 +1 @@
-export * from './DatePicker';
+export { default } from './DatePicker';
```

Comment:
Should we do

```suggestion
export { default } from './DatePicker';
export * from './DatePicker';
```

So developers can import `DatePickerProps`?

Reply:
Doesn't this work in this order?

Reply:
It's not about the order, it's about having two lines instead of one.